distribution/docs/src/main/resources/content/_managing/_configuring/endpoint-utilities-intro.adoc

```diff
@@ -0,0 +1,11 @@
+:title: Endpoint Utility Services
+:type: subConfiguration
+:status: published
+:parent: Configuring Endpoints
+:summary: Endpoint Utility Services
+:order: 06
+
+
+== {title}
+
+${branding} also provides a variety of services to enhance the function of the endpoints.
```

Comment:
❓ Do you anticipate there being more of these? It just looks odd because it talks about a "variety of services", and then there's only one under this section.

Reply:
I thought there were a couple more, let me check on this.

Reply:
I revised it